Sources/SwifterSwift/CoreGraphics/CGRectExtensions.swift

```diff
@@ -0,0 +1,81 @@
+//
+//  CGRectExtensions.swift
+//  SwifterSwift
+//
+//  Created by Chen Qizhi on 2020/03/11.
+//  Copyright © 2020 SwifterSwift
+//
+#if canImport(CoreGraphics)
+import CoreGraphics
+
+// MARK: - Properties
+public extension CGRect {
+
+    /// SwifterSwift: Return center of rect
+    var center: CGPoint {
+        return CGPoint(x: midX, y: midY)
+    }
+
+}
+
+// MARK: - Initializers
+public extension CGRect {
+
+    /// SwifterSwift: Create a `CGRect` instance with center and size
+    /// - Parameters:
+    ///   - center: center of the new rect
+    ///   - size: size of the new rect
+    init(center: CGPoint, size: CGSize) {
+        let origin = CGPoint(x: center.x - size.width / 2.0, y: center.y - size.height / 2.0)
+        self.init(origin: origin, size: size)
+    }
+
+}
+
+// MARK: - Methods
+public extension CGRect {
+
+    /// SwifterSwift: Corners and center of `CGRect`, can be use for resizing,
+    enum Anchor {
+        case topLeft
+        case topRight
+        case bottomLeft
+        case bottomRight
+        case center
+    }
+
+    /// SwifterSwift: Create a new `CGRect` by resizing with specified anchor
+    /// - Parameters:
+    ///   - size: new size to be applied
+    ///   - anchor: specified anchor, excample:
+    ///
+    ///          anchor = .bottomLeft:
+    ///
+    ///                       A2------B2
+    ///          A----B       |        |
+    ///          |    |  -->  |        |
+    ///          C----D       C-------D2
+    ///
+    func resizing(to size: CGSize, anchor: CGRect.Anchor = .topLeft) -> CGRect {
```

Comment:
I think that instead of using the `Anchor` enum it would be better to use a `CGPoint`, just like you have in `CALayer.anchorPoint`. That way users of the function could have more flexibility on where they want to resize from. It can also simplify your code quite a bit:
```Swift
func resizing(to size: CGSize, anchor: CGPoint = CGPoint(x: 0.5, y: 0.5)) -> CGRect {
    let sizeDelta = CGSize(width: size.width - width, height: size.height - height)
    return CGRect(origin: CGPoint(x: minX - sizeDelta.width * anchor.x,
                                  y: minY - sizeDelta.height * anchor.y),
                  size: size)
}
```

Reply:
Great, this is really simple enough and powerful.